Random Users/Helpers/Extensions.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Extensions.swift
+//  Random Users
+//
+//  Created by Jeffrey Santana on 9/6/19.
+//  Copyright © 2019 Erica Sadun. All rights reserved.
+//
+
+import UIKit
+
+extension UIImageView {
+	func loadImage(from url: URL) {
+		DispatchQueue.global().async { [weak self] in
+			if let data = try? Data(contentsOf: url) {
```

Comment:
you'll want to be be using URLSession - `Data(contentsOf:)` (and variants of the like) aren't good for anything aside from testing or mocking. They lock up the main thread if called from there (which you got around, technically) and provide no error handling.

Reply:
updated to using urlsessions